test/instrumentation/modules/ioredis.test.js

```diff
@@ -110,13 +109,16 @@ test('rejections_handled', function (t) {
     process.removeListener('unhandledRejection', onUnhandledRejection)
   })
   agent._instrumentation.testReset()
-  agent._transport = mockClient(3, function () {
+  agent._transport = mockClient(4, function (data) {
+    t.equal(data.errors.length, 1, 'captured 1 error')
+    t.equal(data.errors[0].exception.type, 'ReplyError', 'exception.type')
+    t.equal(data.errors[0].transaction_id, data.transactions[0].id, 'error.transaction_id')
+
     setTimeout(function () {
       t.notOk(unhandledRejection)
       t.end()
     }, 0)
   })
-  agent.captureError = function (err) { throw err }
```

Comment:
REVIEW NOTE: This and the `agent.captureError =` override below were removed because they conflict with testing now that the ioredis instrumentation is actually using `agent.captureError()`.